src/status_im/ui/screens/multiaccounts/key_storage/views.cljs

```diff
@@ -0,0 +1,344 @@
+        (i18n/label :t/seed-key-uid-mismatch-desc-2)]]]
+     [react/view {:margin-vertical 24
+                  :align-items     :center}
+      [quo/button {:on-press            #(re-frame/dispatch [:hide-popover])
+                   :accessibility-label :cancel-custom-seed-phrase
+                   :type                :secondary}
+       (i18n/label :t/try-again)]]]))
+
+(defview transfer-multiaccount-warning-popover []
+  [react/view {:margin-top        24
+               :margin-horizontal 24
+               :align-items       :center}
+   [react/view {:width           32
+                :height          32
+                :border-radius   16
+                :align-items     :center
+                :justify-content :center}
+    [vector-icons/icon :main-icons/tiny-warning-background {:color colors/red}]]
+   [react/text {:style styles/popover-title}
+    (i18n/label :t/move-keystore-file-to-keycard)]
+   [react/view styles/popover-body-container
+    [react/text {:style styles/popover-text}
+     (i18n/label :t/database-reset-warning)]]
+   [react/view {:margin-vertical 24
+                :align-items     :center}
+    [quo/button {:on-press            #(re-frame/dispatch [::multiaccounts.key-storage/delete-multiaccount-and-init-keycard-onboarding])
+                 :accessibility-label :cancel-custom-seed-phrase
+                 :type                :primary
+                 :theme               :negative}
+     (i18n/label :t/move-and-reset)]
+    [quo/button {:on-press            #(re-frame/dispatch [:hide-popover])
+                 :accessibility-label :cancel-custom-seed-phrase
+                 :type                :secondary}
+     (i18n/label :t/cancel)]]])
+
+(defview unknown-error-popover []
+  [react/view {:margin-top        24
+               :margin-horizontal 24
+               :align-items       :center}
+   [react/view {:width           32
+                :height          32
+                :border-radius   16
+                :align-items     :center
+                :justify-content :center}
+    [vector-icons/icon :main-icons/close {:color colors/red}]]
+   [react/text {:style {:typography    :title-bold
+                        :margin-top    8
+                        :margin-bottom 24}}
+    (i18n/label :t/something-went-wrong)]
+   [react/view styles/popover-body-container
+    [react/view
+     [react/text {:style (into styles/popover-text
+                               {:margin-bottom 16})}
+      (i18n/label :t/transfer-ma-unknown-error-desc-1)]
+     [react/text {:style styles/popover-text}
+      (i18n/label :t/transfer-ma-unknown-error-desc-2)]]]
+   [react/view {:margin-vertical 24
+                :align-items     :center}
+    [quo/button {:on-press #(re-frame/dispatch [::multiaccounts.key-storage/hide-popover-and-goto-multiaccounts-screen])
+                 :type     :secondary}
+     (i18n/label :t/okay)]]])
+
+(comment
+  (-> re-frame.db/app-db deref keys)
```

Comment:
if not for test, could we move this in separate ns ?

Reply:
I have deleted db lookups. Re-frisk is much better for tasks like these.